src/rules/at-rule-empty-line-before/__tests__/index.js

```diff
@@ -380,3 +380,163 @@ testRule(rule, mergeTestDescriptions(sharedNeverTests, {
     code: "a {\n\n  color: pink;\n\n  @mixin foo;\n}",
   } ],
 }))
+
+testRule(rule, {
+  ruleName,
+  config: [ "always", { ignoreAtRules: ["else"] } ],
+
+  accept: [ {
+    code: `
+      @if(true) {
+      } @else {
+      }
+    `,
+  }, {
+    code: `
+      @if(true) {
+      }
+
+      @else {
+      }
+    `,
+  }, {
+    code: `
+      @if(true) {}
+      @else if(true) {
+      }  @else {
+      }
+    `,
+  } ],
+
+  reject: [ {
+    code: `
+      @if(true) {
+      } @else-mixin {
+      }
+    `,
+    message: messages.expected,
+    line: 3,
+    column: 9,
+  }, {
+    code: `
+      @if (true) {}
+      @if (false) {
+      }
+    `,
+    message: messages.expected,
+    line: 3,
+    column: 7,
+  } ],
+})
+
+testRule(rule, {
+  ruleName,
+  config: [ "always", { ignoreAtRules: "/fo/" } ],
```

Comment:
Can we use `/el/` (matching `else`) here to avoid the ellipses in the code below?